src/components/navigation/navigation-item/navigation-item.component.html

```diff
@@ -0,0 +1,7 @@
+<a *ngIf="link" [class.has-arrow]="children.length > 0" [class.no-arrow]="indentWithoutArrow()" [routerLink]="link">
```

Comment:
Is it possible to determine in code if it should have no arrow and store the result in a variable instead of calling `indentWithoutArrow()` in the view? This would result in this function being called every time change detection occurs which is quite inefficient.

Reply:
Perhaps an observable based on `this._children.changes`?

Reply:
Information now stored in variable which is assigned a value in ngAfterContentInit().